add_gif.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Usage: ./add_gif.sh name http://url_of_the_gif
+NAME=$1
+URL=$2
+
+# Usage 
+if [ "$NAME" = "--help" ]; then
```

Comment:
When using bash, you should prefer the bash test built-in `[[`. How about we print help if the arg count is off:
`[[ "$#" -ne 3 ]]`